### PR TITLE
replace _ by - using sed

### PR DESCRIPTION
**Pull request by daschuer**
For some reasons `"${lang/_/-}"``fails on the build runner. 
`build_html.sh: 27: Bad substitution`
My local bash did not suffer this issue,  ... strange. 
Here is a version using sed

### CODE REVIEW

File: build_html.sh [hunk 1]
```diff
@@ -24,7 +24,7 @@ do
         printf '/:version/*    /:version/en/:splat    301\n' >> build/html/_redirects
     else
         printf '/:version/%s/* /:version/%s/404.html  404\n' "$lang" "$lang" >> build/html/_redirects
-        printf '/:version/*    /:version/%s/:splat    301 Language=%s\n' "$lang" "${lang/_/-}" >> build/html/_redirects
+        printf '/:version/*    /:version/%s/:splat    301 Language=%s\n' "$lang" "`echo "$lang"  | sed 's/_/-/g'`" >> build/html/_redirects
```
Comment (Holzhaus) [inline build_html.sh:27]:
```suggestion
        printf '/:version/*    /:version/%s/:splat    301 Language=%s\n' "$lang" "$(printf "%s" "$lang"  | sed 's/_/-/g')" >> build/html/_redirects
```
Printf is a shell builtin (in contrast to echo) and `$()` is more readable than legacy backtick notstion IMHO.

**Comment by daschuer** [comment]
Done and rebased.

**Comment by daschuer** [comment]
Cool, the adjustment for linkcheck did the trick.

**Comment by Holzhaus** [comment]
Thanks!

**Pull request merged**